setup.py

```diff
@@ -19,10 +19,18 @@
 
 
 def _get_cpu_count():
-    if platform.system() != "Windows":
-        return os.cpu_count()
-    return 0
+    if platform.system() == "Windows":
+        return 0
 
+    cpu_count = os.cpu_count()
+    try:
+        user_max_cores = int(os.getenv('MAX_BUILD_CORES', cpu_count))
+    except ValueError:
+        raise ValueError(
```

Comment:
So, while this looks fine, I think it’s not the best way to reraise an exception, see https://stackoverflow.com/questions/9157210/how-do-i-raise-the-same-exception-with-a-custom-message-in-python

Reply:
I read through that originally when I was looking for a better way, but I missed the solution presented lower down. The change to the resulting error message is pretty minor, but better. The chances of the user ending up here seem pretty low.